solutions/fully-configurable/version.tf

```diff
@@ -0,0 +1,11 @@
+terraform {
+  required_version = ">=1.9.0"
+
+  # Lock DA into an exact provider version - renovate automation will keep it updated
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = "1.75.2"
```

Comment:
1.76.1 has a fix for auth policy updates, so I would update to that or later (1.76.2 was just released)